### PR TITLE
One direction of sb5 is provable from fewer axioms

### PR DESCRIPTION
**Pull request by wlammen**
1. One direction of sb5 is provable from axioms up to ax-7 only.  Add this theorem as sb1v, matching the similar sb1 in name.
2. Delete outdated OLD theorems

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -541596,6 +541577,14 @@ dependency is expressed in our hypothesis (called implicit
       UGCBUGCLMABCNOUCUEABUECBHAICJZUCAACBPUCUHAACBNQRSTUA $.
   $}
 
+  ${
+    $d x y $.
+    $( One direction of ~ sb5 is provable from fewer axioms.  See also ~ sb1 .
+       (Contributed by Wolf Lammen, 20-Jan-2024.) $)
```
Comment (benjub) [inline set.mm:541583]:
```suggestion
    $( One direction of ~ sb5 , provable from fewer axioms.  Version of
        ~ sb1 with disjoint variables, requiring fewer axioms.
       (Contributed by Wolf Lammen, 20-Jan-2024.) $)
```

**Comment by benjub** [comment]
I think wl-sb1v should be moved to Main. Can it be used in other theorems (possibly adapting the proofs), in particularly the ones using sb1/sb5 ?

**Review by benjub — APPROVED** [review]

**Comment by wlammen** [comment]
`Can it be used in other theorems (possibly adapting the proofs), in particularly the ones using sb1/sb5 ?`

I did a quick check to see whether it helps in other theorems, but I found no instance.

**Comment by benjub** [comment]
> `Can it be used in other theorems (possibly adapting the proofs), in particularly the ones using sb1/sb5 ?`
> 
> I did a quick check to see whether it helps in other theorems, but I found no instance.

To me, even without applications, it's worth it in its own sake. So, now or in a future PR, as you prefer.

**Review by icecream17 — APPROVED** [review]

**Review by tirix — APPROVED** [review]

File: set.mm [hunk 2]
```diff
@@ -229301,17 +229293,6 @@ U C_ ( T .(+) U ) ) $=
       ( csubg cfv wcel csubmnd co wceq subgsubm smndlsmidm syl ) BCEFGBCHFGBBAI
       BJBCKABCDLM $.
 
-    $( Obsolete proof of ~ lsmidm as of 13-Jan-2023.  Subgroup sum is
```
Comment (avekens) [inline set.mm:229304]: Sorry, my fault: I replaced the proof in one of my latest commits, so the date should have been 13-Jan-2024. Could you add this theorem again (with corrected date), please?

**Review by avekens — APPROVED** [review]

**Pull request merged**